core/checker.go

```diff
@@ -240,7 +256,8 @@ func (s *Service) checkHttp(record bool) *Service {
 	}
 	if s.ExpectedStatus != res.StatusCode {
 		if record {
-			recordFailure(s, fmt.Sprintf("HTTP Status Code %v did not match %v", res.StatusCode, s.ExpectedStatus))
+			recordFailure(s, fmt.Sprintf("HTTP Status Code %v did not match %v",
```

Comment:
```suggestion
			recordFailure(s, fmt.Sprintf("HTTP status code %v did not match %v",
```

This is really only for uniformity. Things tend to look and feel alot better when they're uniform in appearance in my opinion. I suggest revisiting all comments and have them follow standard grammatical practices with regards to caps - of course, special words/expressions do have their place and are meaningful